forte/data/data_store.py

```diff
@@ -0,0 +1,231 @@
+from typing import Union, Type, List, Iterator
+
+import uuid
+from forte.data.ontology.core import EntryType
+from forte.data.base_store import BaseStore
+
+
+class DataStore(BaseStore):
+    def __init__(self):
+        """An implementation from the dataframe-like data tuple object.
+
+                This class stores entries by types in different lists. It
```

Comment:
Is the indention here done on purpose?

Reply:
I used black to format it and somehow it just became like this. I will remove the indentation.

Reply:
To confirm if the documentation is correct, need to build the doc locally to see if the output is good: https://github.com/asyml/forte/blob/master/.github/workflows/main.yml#L137

Reply:
sounds good!